### ISSUE TITLE
Style of Translations

### ISSUE THREAD

**Issue by clzls** [microsoft/winfile#151]
Should we follow Microsoft Style Guide strictly in the translations? 
We didn't follow the official Microsoft voice strictly in current version of Simplified Chinese translations in ``res_zh-CN.rc`` (since it's my native language so I can only check this one). As we are working on more languages (Japanese, Polish, Hebrew), we should set up a standard.

**Comment by peterooch** [comment]
https://www.microsoft.com/en-us/language/StyleGuides

**Comment by ZanderBrown** [comment]
In terms of language probably except where it departs for the current style, does anyone have the 3x era equivalent of this document? (Was there one?)

**Comment by peterooch** [comment]
One obvious example is "directory", in Hebrew 3.11 it was translated to the word equivalent to "library".
the term "library" means mostly (in computer terminology) software libraries.
Since then it got replaced by "folder" which is way more clear in the meaning.

**Comment by wbsdty331** [comment]
Maybe we could use Fluent Design style to redesign this file manager :-)

**Comment by ZanderBrown** [comment]
@wbsdty331  No?

**Comment by artursouza** [comment]
TL;DR: No redesign, please.

I see this as a retro tool. Redesigning would remove its main purpose to
exist, IMO.

**Comment by fitojb** [comment]
Note that not all language guides have underwent positive changes. Microsoft’s Spanish translations have gone really bad since Windows 8 (more unnecessary terminology calques instead of patrimonial equivalents, abuse of the familiar _tú_ treatment, disregard of native punctuation rules…)

**Comment by sr229** [comment]
I think we should follow translation style as we did on VSCode. Its inherently better since we have people working over it.

**Comment by clzls** [comment]
It seems to be settled and maybe not so meaningful now since only some minor bugs left and few would like a totally rebuilding of the all these translations (a waste of time, right?).
I think it may be the time to close this issue.
